Assets/Scripts/Models/Area/World.cs

```diff
@@ -212,6 +215,34 @@ public void InvalidateTileGraph()
         tileGraph = null;
     }
 
+    public Path_TileGraph GetTileGraph()
+    {
+        if (tileGraph == null)
+        {
+            tileGraph = new Path_TileGraph(this);
+        }
+
+        return tileGraph;
+    }
+
+    public void RegenerateGraphAtTile(Tile tile)
+    {
+        if (tileGraph != null)
+        {
+            tileGraph.RegenerateGraphAtTile(tile);
+        }
+    }
+
+    public Path_RoomGraph GetRoomGraph()
```

Comment:
Perhaps these should be properties rather than Get functions.

Reply:
Will think on it...